scripts/publish-cranelift.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
+
+# This is a convenience script for maintainers publishing a new version of
+# Cranelift to crates.io. To use, first bump the version number by running the
+# `scripts/cranelift-version.sh` script, then run this script, and run the
+# commands that it prints.
+#
+# Don't forget to push a git tag for this release!
```

Comment:
The previous script did print on the terminal the instructions to run for the `git tag` et al., is this something we'd like to keep on doing here?